amplify/src/main/java/com/github/stkent/amplify/tracking/GenericSettings.java

```diff
@@ -79,4 +79,8 @@ public T getEventValue(@NonNull final IEvent event) {
         return null;
     }
 
+    public boolean hasEventValue(@NonNull final IEvent event) {
+        return sharedPreferences.contains(event.getTrackingKey());
```

Comment:
Hmm that was one of the things my refactor removed... Let me think on this one and how to get that back. Any ideas how we should adjust this?